app/assets/javascripts/Components/ta_table.jsx

```diff
@@ -36,6 +36,13 @@ class TATable extends React.Component {
       });
   }
 
+  removeTA = ta_id => {
+    $.ajax({
```

Comment:
This is correct, but for new methods prefer using `fetch` instead of `$.ajax` (we're slowly trying to move away from jQuery entirely).

See https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch, and note you can pass a `method` configuration option